requirements.test.txt

```diff
@@ -1,21 +1,16 @@
 #Install the agent
 .
 # app
-f5-sdk
 git+https://github.com/openstack/neutron#stable/liberty
 git+https://github.com/openstack/oslo.log.git@stable/liberty
 git+https://github.com/F5Networks/f5-openstack-lbaasv2-driver.git@liberty
+git+https://github.com/F5Networks/f5-openstack-test.git
 git+https://github.com/openstack/neutron-lbaas.git@stable/liberty
 
-# Test Requirements
-# git+ssh://git@bldr-git.int.lineratesystems.com/tools/pytest-meta.git
-# git+ssh://git@bldr-git.int.lineratesystems.com/tools/pytest-symbols.git
-git+https://github.com/F5Networks/f5-openstack-test.git@liberty
-git+https://github.com/F5Networks/pytest-symbols.git
-
-mock==1.3.0
-pytest==2.9.1
+f5-sdk==1.5.0
+mock==2.0.0
+pytest==2.9.2
 pytest-cov==2.3.1
-responses
-coverage==4.2
-python-coveralls==2.7.0
+responses==0.5.1
+coverage==4.1
+python-coveralls==1.1
```

Comment:
Why did you downgrade the the last two requirements?  This seems unrelated to a buildbot integration and may break how we collect code coverage and upload to coveralls.